ci/docker/data.build.wanda.yaml

```diff
@@ -11,5 +11,6 @@ srcs:
   - python/requirements/ml/data-test-requirements.txt
 build_args:
   - DOCKER_IMAGE_BASE_BUILD=cr.ray.io/rayproject/oss-ci-base_ml-py$PYTHON
+  - ARROW_VERSION=16.*
```

Comment:
what does the `*` in `16.*` mean?

Reply:
i means 16 as the major version and anything as minor or patch version